geodesy/package.xml

```diff
@@ -25,17 +26,24 @@
 
   <run_depend>geographic_msgs</run_depend>
   <run_depend>geometry_msgs</run_depend>
-  <run_depend>python-pyproj</run_depend>
   <run_depend>tf</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>unique_id</run_depend>
   <run_depend>uuid_msgs</run_depend>
 
+  <run_depend condition="$ROS_PYTHON_VERSION == 2">python-pyproj</run_depend>
+  <run_depend condition="$ROS_PYTHON_VERSION == 3">python3-pyproj</run_depend>
```

Comment:
This is fine once a `python3-pyproj` rosdep key exists. Would you be willing to open a PR on `ros/rosdistro`?

Reply:
Already done: https://github.com/ros/rosdistro/pull/24114. I also added arch which was missing